ixmp/tests/reporting/test_reporting.py

```diff
@@ -491,7 +491,7 @@ def test_reporter_describe(test_mp, test_data_path, capsys):
     - <ixmp.core.Scenario object at {id}>
   - 'config':
     - {{'filters': {{}}}}""".format(id=id_)
-    assert desc1 == r.describe('d:i')
+    assert desc1 == r.describe('d:i', quiet=False)
```

Comment:
Does this actually need to change, or will it still work with the default (quiet=True)?